scripts/check_e2e_tests_are_captured_in_ci.py

```diff
@@ -29,7 +29,7 @@
 # One is extra (ie. (full: [*.js])) and three other test suites are
 # are being run by CircleCI.
 TEST_SUITES_NOT_RUN_ON_TRAVIS = [
-    'full', 'accessibility', 'adminPage', 'classroomPage',
+    'full', 'adminMiscTab', 'adminPage', 'accessibility', 'classroomPage',
```

Comment:
Keep this in alphabetical order (except for `full`) please

Reply:
changed!